test/indexer/data/aaa96233-bf27-44c7-82df-b4dc15ad4d9d.2018-11-02T113344.698028Z.results.json

```diff
@@ -279,6 +278,73 @@
                 ],
                 "matrices": [],
                 "contributor_matrices": [],
+                "aggregate_dates": [
```

Comment:
I stated multiple times during our conversation that we need to pre-aggregate these to keep the contributions small. There should only be single item in this list.